itl/plugins-contrib.d/smart-attributes.conf

```diff
@@ -0,0 +1,15 @@
+/* 
+ * Icinga2 CheckCommand definition for the SMART Attributes Monitoring Plugin
+ */
+
+object CheckCommand "smart-attributes" {
+   import "plugin-check-command"
+   command = [ PluginDir + "/check_smart_attributes" ]
+   arguments = {
+   "-dbj" = "/etc/nagios-plugins/config/check_smartdb.json"
```

Comment:
Thomas added the configurable path in this commit cadc7afd32408c864267c5bc276fae6e24119e1e - does this seem ok?